bin/util.py

```diff
@@ -550,18 +559,32 @@ def strip_newline(s):
     else:
         return s
 
+# check if windows supports symlinks
+if is_windows() or True:
+    link_parent = Path(tempfile.gettempdir()) / 'bapctools'
+    link_dest = link_parent / 'dir'
+    link_dest.mkdir(parents=True, exist_ok=True)
+    link_src = link_parent / 'link'
+    if link_src.exists() or link_src.is_symlink():
+        link_src.unlink()
+    try:
+        link_src.symlink_to(link_dest, True)
+        windows_can_symlink = True
+    except:
+        windows_can_symlink = False
+        warn('Please enable the developer mode in Windows!')
```

Comment:
`... to enable symlinks`?
And add a link to instructions on how to do this. This warning should be rare so may as well intruct on how to fix.